schemas/SDMXStructureDataflow.xsd

```diff
@@ -25,19 +25,31 @@
 							<xs:documentation>Structure provides a reference to the data structure definition that defines the structure of all data for this flow.</xs:documentation>
 						</xs:annotation>
 					</xs:element>
-					<xs:element name="DimensionConstraint" type="common:DimensionConstraintReferenceType" minOccurs="0">

```

Comment:
Cannot find the `common:DimensionConstraintReferenceType` definition. Should be in SDMXCommonReferences.xsd

Reply:
it is in SDMXCommonReferences.xsd on line 1314